src/scss/grommet-core/_objects.search-input.scss

```diff
@@ -1,53 +1,54 @@
 // (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
 
+.grommet .search-input,
```

Comment:
Why do we need the .grommet prefix here? I was expecting we only needed it around base DOM elements, like 'a' and 'button'.